client/my-sites/plugins/mailpoet-confirmation/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { Card, Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import Main from 'calypso/components/main';
+import { addMailPoetUpgrade } from 'calypso/data/marketplace/mailpoet-add-upgrade';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
+import './style.scss';
+
+export const MailPoetConfirmationPage = ( { siteId }: { siteId: number } ) => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+	const [ isBusy, setIsBusy ] = useState( false );
+	const [ isCompleted, setIsCompleted ] = useState( false );
+
+	const getItNowClickHandler = async () => {
+		setIsBusy( true );
+		try {
+			await addMailPoetUpgrade( siteId );
+
+			dispatch(
+				successNotice(
+					translate(
+						'The MailPoet is added to your site. MailPoet plugin will be installed and activated shortly.'
+					)
+				)
+			);
+		} catch ( error ) {
+			dispatch( errorNotice( ( error as Error ).message ) );
+		} finally {
+			setIsBusy( false );
+			// Either error or success. We will mark it as completed so user can navigate back
+			setIsCompleted( true );
+		}
+	};
+
+	const dismissHandler = () => {
+		window.history.back();
+	};
+
+	return (
+		<Main wideLayout className="mailpoet-confirmation__main mailpoet-confirmation__no-header">
+			<DocumentHead title={ translate( 'MailPoet Subscription' ) } />
+			<FormattedHeader
+				brandFont
+				headerText={ translate( 'MailPoet Subscription' ) }
+				subHeaderText={ translate( 'Get a Free MailPoet Subscription' ) }
+				align="left"
+				hasScreenOptions
+			/>
+			<Card>
+				<p>
+					{ translate(
+						'Your eCommerce plan provides a complimentary MailPoet Business subscription, allowing you to send visually appealing emails that consistently land in inboxes and cultivate a loyal subscriber base.'
```

Comment:
```suggestion
						'Your Commerce plan provides a complimentary MailPoet Business subscription, allowing you to send visually appealing emails that consistently land in inboxes and cultivate a loyal subscriber base.'
```

I think we've renamed the eCommerce plan to just Commerce?

Reply:
That's correct! Changing it 
![CleanShot 2023-09-27 at 13 53 47@2x](https://github.com/Automattic/wp-calypso/assets/3519124/7c425525-9b98-47f5-90ea-a198221848b7)

Reply:
Changed in 019b1b5a1bc223848ae169124e6c2a675dcc2ec